src/Hartree_CG.h

```diff
@@ -40,6 +40,13 @@ class Hartree_CG : public Poisson
 
     void solve(const pb::GridFunc<RHODTYPE>& rho,
         const pb::GridFunc<RHODTYPE>& rhoc) override;
+
+    void applyOperator(pb::GridFunc<POTDTYPE> &vh,
```

Comment:
This does not look natural to me. Hartree_CG is a solver, that uses an operator. If you just need the operator, just create it instead of getting it out of that class.

Reply:
- ROM solver wants to use the same operator that FOM solver uses. So it is natural to take it from the FOM solver. Please see the `testRoMPoisson` routine for how ROM 'mirrors' FOM solver.
- Eventually we would want to support all operators. If we read the `Controls` again and create the operator again, then we'll have to duplicate the long switch statement in `Electrostatic.cc`, which is just error-prone and more complicated. By doing this, we can already use at least 4 different FD operators (Lap2, 4, 6, 8) for ROM, without any switch statement.

Reply:
As discussed: MGmol design could be better to accommodate this use case, but in the interest of time, let's merge it as is.